google/cloud/storage/tests/object_integration_test.cc

```diff
@@ -898,8 +898,6 @@ TEST_F(ObjectIntegrationTest, DeleteAccessControlFailure) {
 }
 
 TEST_F(ObjectIntegrationTest, DeleteResumableUpload) {
-  // TODO(#5910) - DeleteResumableUpload doesn't work with gRPC yet.
-  if (UsingGrpc()) GTEST_SKIP();
```

Comment:
Note that some of these RPCs are only implemented in the emulator for gRPC, production only implements the basic upload and download operations at the moment.

Reply:
Fixed.